docker-compose.yml

```diff
@@ -2,35 +2,26 @@ version: '3'
 
 services:
   algod:
-    image: "sdk-harness-algod"
+    image: algorand/algorand-sdk-testing-algod:latest
```

Comment:
If I need to update these dockerfiles, what is the process for testing the environment locally?

Reply:
To do that you just need to run this command

```
mule -f cicd/mule.yaml build-local-images
```